setup.py

```diff
@@ -1,7 +1,5 @@
-import distutils.command
-import distutils.command.build_ext
-import distutils.core
-import distutils.errors
+from setuptools import setup
+from setuptools.extension import Extension
```

Comment:
Ideally the script would still work if setuptools (which isn't part of the standard library yet) isn't available. It should be as simple as trying to import setuptools and if not present, falling back on the distutils functions of the same name...